antiplag/nlp/text_preprocessing.py

```diff
@@ -14,7 +14,20 @@
 import string
 from pycontractions import Contractions
 from textblob import TextBlob
+import textract
+import os
 
+def text_extraction(path_to_file):
+    file_name, file_extension = os.path.splitext(path_to_file)
+
+    if file_extension == '.srt' or file_extension == '.md':
+        file_extension = '.txt'
+        path_to_file_old = path_to_file
+        path_to_file = file_name + file_extension
+        os.rename(path_to_file_old, path_to_file)
```

Comment:
Why do we change file names when we only want to extract the text?

Reply:
Seems like it's a little complicated - python [textract](https://github.com/deanmalmgren/textract) does not support `srt` and `markdown` files - therefore we move the content of the files into `txt` file which is later processed by `textract`.

I contacted Juro with this thing whether we should support this type of files or not.